dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZKServer.java

```diff
@@ -108,31 +131,29 @@ public static void startLocalZkServer(final int port) {
      * @param tickTime    zk tick time
      * @param maxClientCnxns    zk max client connections
      */
-    private static synchronized void startLocalZkServer(final int port, final String dataDirPath,final int tickTime,String maxClientCnxns) {
-        if (zkServer != null) {
-            throw new RuntimeException("Zookeeper server is already started!");
-        }
-        zkServer = new PublicZooKeeperServerMain();
-        logger.info("Zookeeper data path : {} ", dataDirPath);
-        dataDir = dataDirPath;
-        final String[] args = new String[]{Integer.toString(port), dataDirPath, Integer.toString(tickTime), maxClientCnxns};
+    private void startLocalZkServer(final int port, final String dataDirPath,final int tickTime,String maxClientCnxns) {
```

Comment:
```suggestion
    private void startLocalZkServer(final int port, final String dataDirPath, final int tickTime, String maxClientCnxns) {
```
Hi,
Thx a lot for your good job and contribution.
Please configure the `checkstyle.xml` plugin[1] to your IDE to format code.
[1] https://github.com/apache/incubator-dolphinscheduler/tree/dev/style